docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -822,6 +828,40 @@
                 </tr>
 
                 <tr>
+                    <th colspan="3" class="section" id="streams"><a href="../container/streams.html">Streams</a></th>
+                </tr>
+
+                <tr>
+                    <td class="property" id="streams-system">streams.<span class="stream">stream-id</span>.samza.system</td>
```

Comment:
In the past, streamName was used as both:
1. a logical identifier in samza. e.g. to look up stream properties in the config
2. a physical identifier in the System e.g. as the topic name for Kafka

StreamId and physicalName represent the separation of those two roles and provide the following benefits:
1. StreamId is now a single unique identifier for a stream. A streams ID is no longer coupled to a System, so it's more portable and less verbose.
2. A streamID does not allow special characters so we can guarantee config parsing, etc. Whereas physicalName does, and that enables physical identifiers that have special characters like an HDFS file URN